cob_hardware_config/robots/cob4-10/config/image_flip_params_sensorring_cam3d_front.yaml

```diff
@@ -12,7 +12,7 @@ reference_frame: base_link
 
 # flip color image
 # bool
-flip_color_image: true
+flip_color_image: false
```

Comment:
why false?

Reply:
sick_3dcs does not have a color_image...@ipa-rmb @ipa-josh

Reply:
ok fine